.lando.yml

```diff
@@ -1,7 +1,7 @@
 name: moose
 recipe: wordpress
 config:
-  php: '8.0'
+  php: '8.1'
```

Comment:
Most of the current hosts are still at 8.0. I am glad we are set up to run in 8.1, but I don't think we can default to it yet. Is it a requirement for `block-editor-custom-alignments`?

Reply:
No - not a requirement for my plugin. Something else got updated when I reinstalled composer packages. I was getting a fatal PHP error that said 8.1 was required. See my comment above that has a bit more information.

Reply:
Cool, I can take a look at that.